beacon-chain/p2p/encoder/ssz.go

```diff
@@ -16,7 +16,8 @@ import (
 var _ NetworkEncoding = (*SszNetworkEncoder)(nil)
 
 // MaxGossipSize allowed for gossip messages.
-var MaxGossipSize = params.BeaconNetworkConfig().GossipMaxSize // 1 Mib
+var MaxGossipSize = params.BeaconNetworkConfig().GossipMaxSize // 1 Mib.
```

Comment:
Lets unexport these vars and keep their scope in this package.

Reply:
as discussed in discord, its hard to do this since we use the global var in test cases (with a different package)